app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,70 @@
+class RentalsController < ApplicationController
+
+  before_action :find_customer
+  before_action :find_video
+
+  def check_out
```

Comment:
These methods are a bit long, maybe creating a check-out and check-in methods for the model-level.